packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -2207,6 +2334,127 @@ curl -v -X POST \
 }
 ```
 
+#### Activate WebAuthn Factor
+
+Activation gets the registration information from the WebAuthn authenticator using the API and passes it to Okta.
+
+##### Get registration information from WebAuthn authenticator by calling the WebAuthn Javascript API
+
+```html
+<!-- Using CryptoUtil.js from https://github.com/okta/okta-signin-widget/blob/master/src/util/CryptoUtil.js -->
+<script>
+// Convert activation object's challenge and user id from string to binary
+response._embedded.activation.challenge = CryptoUtil.strToBin(response._embedded.activation.challenge);
+response._embedded.activation.user.id = CryptoUtil.strToBin(response._embedded.activation.user.id);
+
+// navigator.credentials is a global object on WebAuthn-supported clients, used to access WebAuthn API
+navigator.credentials.create({
+  publicKey: response._embedded.activation
+})
+  .then(function (newCredential) {
+    // Get attestation and clientData from callback result, convert from binary to string
+    var attestation = CryptoUtil.binToStr(newCredential.response.attestationObject);
+    var clientData = CryptoUtil.binToStr(newCredential.response.clientDataJSON);
+  })
+  .fail(function (error) {
+    // Error from WebAuthn platform
+  });
+</script>
+```
+
+Activate a WebAuthn factor by verifying the attestation and client data.
+
+##### Activate WebAuthn Request Parameters
+
+
+| Parameter         | Description                                                     | Param Type | DataType | Required |
+| ----------------- | --------------------------------------------------------------- | ---------- | -------- | -------- |
+| factorId          | `id` of factor returned from enrollment                         | URL        | String   | TRUE     |
+| stateToken        | [state token](#state-token) for current transaction             | Body       | String   | TRUE     |
+| attestation       | base64 encoded attestation from WebAuthn javascript call        | Body       | String   | TRUE     |
```

Comment:
adding the word the and hyphenating base64 encoded:

base64-encoded attestation from the WebAuthn javascript call